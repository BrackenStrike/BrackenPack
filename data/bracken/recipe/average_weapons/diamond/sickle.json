{
	"type": "minecraft:smithing_transform",
	"template": "minecraft:host_armor_trim_smithing_template",
	"base": "minecraft:diamond_sword",
	"addition": "minecraft:diamond",
	"result": {
		"id": "minecraft:diamond_sword",
		"components": {
			"minecraft:item_name": "Diamond Sickle",
			"minecraft:item_model": "bracken:average_weapons/diamond/sickle",
			"minecraft:lore": [
				{"translate":"","color":"white","italic":false},
				{"translate":"When in Main Hand:","color":"gray","italic":false},
				{"translate": "8 Attack Damage","color":"dark_green","italic":false},
				{"translate": "1.6 Attack Speed","color":"dark_green","italic":false}
			],
			"minecraft:attribute_modifiers":[{
					"type": "minecraft:attack_damage",
					"id": "bracken:diamond_sickle",
					"amount": 7,
					"operation": "add_value",
					"slot": "mainhand"
				},{
					"type": "minecraft:attack_speed",
					"id": "bracken:diamond_sickle",
					"amount": -2.4,
					"operation": "add_value",
					"slot": "mainhand"
				}],
				"minecraft:tooltip_display": {
        "hidden_components": [
          "minecraft:attribute_modifiers"
        ]
      }
		}
	}
}