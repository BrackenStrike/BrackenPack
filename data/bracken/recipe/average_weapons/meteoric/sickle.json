{
	"type": "minecraft:smithing_transform",
	"template": "minecraft:host_armor_trim_smithing_template",
	"base": "minecraft:netherite_sword",
	"addition": "minecraft:netherite_scrap",
	"result": {
		"id": "minecraft:netherite_sword",
		"components": {
			"minecraft:item_name": "\"Meteoric Sickle\"",
			"minecraft:item_model": "bracken:average_weapons/meteoric/sickle",
			"minecraft:lore": [
				"{\"translate\":\"\",\"color\":white,\"italic\":false}",
				"{\"translate\":\"When in Main Hand:\",\"color\":gray,\"italic\":false}",
				"{\"translate\":\" 9 Attack Damage\",\"color\":dark_green,\"italic\":false}",
				"{\"translate\":\" 1 Attack Speed\",\"color\":dark_green,\"italic\":false}"
			],
			"minecraft:attribute_modifiers": {
				"modifiers": [{
					"type": "minecraft:attack_damage",
					"id": "bracken:meteoric_sickle",
					"amount": 8,
					"operation": "add_value",
					"slot": "mainhand"
				},{
					"type": "minecraft:attack_speed",
					"id": "bracken:meteoric_sickle",
					"amount": -3,
					"operation": "add_value",
					"slot": "mainhand"
				}],
				"show_in_tooltip": false
			}
		}
	}
}