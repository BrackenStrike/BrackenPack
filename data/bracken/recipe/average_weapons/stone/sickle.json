{
	"type": "minecraft:smithing_transform",
	"template": "minecraft:host_armor_trim_smithing_template",
	"base": "minecraft:stone_sword",
	"addition": "#minecraft:stone_tool_materials",
	"result": {
		"id": "minecraft:stone_sword",
		"components": {
			"minecraft:item_name": "Stone Sickle",
			"minecraft:item_model": "bracken:average_weapons/stone/sickle",
			"minecraft:lore": [
				{"translate":"","color":"white","italic":false},
				{"translate":"When in Main Hand:","color":"gray","italic":false},
				{"translate": "7 Attack Damage","color":"dark_green","italic":false},
				{"translate": "1.6 Attack Speed","color":"dark_green","italic":false}
			],
			"minecraft:attribute_modifiers":[{
					"type": "minecraft:attack_damage",
					"id": "bracken:stone_sickle",
					"amount": 6,
					"operation": "add_value",
					"slot": "mainhand"
				},{
					"type": "minecraft:attack_speed",
					"id": "bracken:stone_sickle",
					"amount": -2.4,
					"operation": "add_value",
					"slot": "mainhand"
				}],
				"minecraft:tooltip_display": {
        "hidden_components": [
          "minecraft:attribute_modifiers"
        ]
      }
		}
	}
}