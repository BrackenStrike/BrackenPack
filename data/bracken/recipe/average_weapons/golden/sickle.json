{
	"type": "minecraft:smithing_transform",
	"template": "minecraft:host_armor_trim_smithing_template",
	"base": "minecraft:golden_sword",
	"addition": "minecraft:gold_ingot",
	"result": {
		"id": "minecraft:golden_sword",
		"components": {
			"minecraft:item_name": "\"Golden Sickle\"",
			"minecraft:item_model": "bracken:average_weapons/golden/sickle",
			"minecraft:lore": [
				"{\"translate\":\"\",\"color\":white,\"italic\":false}",
				"{\"translate\":\"When in Main Hand:\",\"color\":gray,\"italic\":false}",
				"{\"translate\":\" 5 Attack Damage\",\"color\":dark_green,\"italic\":false}",
				"{\"translate\":\" 1.6 Attack Speed\",\"color\":dark_green,\"italic\":false}"
			],
			"minecraft:attribute_modifiers": {
				"modifiers": [{
					"type": "minecraft:attack_damage",
					"id": "bracken:golden_sickle",
					"amount": 4,
					"operation": "add_value",
					"slot": "mainhand"
				},{
					"type": "minecraft:attack_speed",
					"id": "bracken:golden_sickle",
					"amount": -2.4,
					"operation": "add_value",
					"slot": "mainhand"
				}],
				"show_in_tooltip": false
			}
		}
	}
}